src/components/AppBar/AppBar.jsx

```diff
@@ -40,10 +42,17 @@ const AppBar = () => {
       <div className='app-bar__container'>
         <div className='app-bar__content container'>
           <Link className='app-bar__logo' to={ROUTES_PATHS.home}>
-            <Icon
-              iconClass='app-bar__logo-img'
-              idIcon={isDark ? 'icon-app-logo-dark' : 'icon-app-logo-light'}
-            />
+            {isDesktop ? (
+              <Icon
+                iconClass='app-bar__logo-img'
+                idIcon={isDark ? 'icon-app-logo-dark' : 'icon-app-logo-light'}
+              />
+            ) : (
+              <Icon
+                iconClass='app-bar__logo-img'
+                idIcon={isDark ? 'icon-logo-mobile-light' : 'icon-logo-mobile-dark'}
+              />
+            )}
```

Comment:
made idIcon variable with use memo instead of this ternary

Reply:
fixed